File: src/Presentation/Mintada.Web/src/models/RulerIssuerCoinTypeGroupDto.ts
import type { RulerCoinTypeItemDto } from './RulerCoinTypeItemDto';

export type RulerIssuerCoinTypeGroupDto = {
  issuerId: number;
  issuerName?: string | null;
  issuerUrlSlug?: string | null;
  territoryType?: string | null;
  coinTypes: RulerCoinTypeItemDto[];
};

File: src/Presentation/Mintada.Api/Dtos/RulerIssuerCoinTypeGroupDto.cs
namespace Mintada.Api.Dtos;

public class RulerIssuerCoinTypeGroupDto
{
    public int IssuerId { get; set; }
    public string? IssuerName { get; set; }
    public string? IssuerUrlSlug { get; set; }
    public string? TerritoryType { get; set; }

    public List<RulerCoinTypeItemDto> CoinTypes { get; set; } = new();
}
